clouddq/classes/dataplex_entity_schema_field.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""todo: add classes docstring."""
+from __future__ import annotations
+
+from dataclasses import dataclass, asdict
+from clouddq.utils import assert_not_none_or_empty
+
+@dataclass
+class DataplexEntitySchemaField:
+    """ """
+
+    name: str
+    data_type: str
+    mode: str
+
+    def to_dict(self: DataplexEntitySchemaField) -> dict:
+        """
+        Args:
+          self: DataplexEntitySchemaField:
+
+        Returns:
+
+        """
+
+        return asdict(self)
+
+
+    @classmethod
+    def from_dict(cls: DataplexEntitySchemaField, kwargs: dict) -> DataplexEntitySchemaField:
```

Comment:
where is this called?

Reply:
I created this for testing purpose, if we want to mock schema for the dataplex entity.
Not getting called anywhere.